### PR TITLE
Feature/create manual model

### LINKED ISSUE #7: Enable the backend to save, edit and publish manuals

### PR DESCRIPTION
**Pull request by WanjikuMac**
Fixes #7 
## Proposed Changes

  - Create the manual model
  -
  -

### CODE REVIEW

File: test/lib/qb_backend/Posts/manual_test.exs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+defmodule QbBackend.Posts.ManualTest do
+  @moduledoc """
+  This module holds tests for the manual changesets
+  """
+  use QbBackend.DataCase
+
+  alias QbBackend.{
+    Posts.Manual,
+    Posts.Profile
```
Comment (zacck-zz) [inline test/lib/qb_backend/Posts/manual_test.exs]: is this used? I think it should be Accounts.Profile if it is

**Review by zacck-zz — COMMENTED** [review]
Hey @WanjikuMac  good swift work! I think a little change to push this over the hill

File: test/lib/qb_backend/posts/manual_test.exs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+defmodule QbBackend.Posts.ManualTest do
+  @moduledoc """
+  This module holds tests for the manual changesets
+  """
+  use QbBackend.DataCase
+
+  alias QbBackend.{
+    Posts.Manual,
+    Posts.Profile
```
Comment (zacck-zz) [inline test/lib/qb_backend/posts/manual_test.exs]: @br4inii  this is unused and can be removed

**Review by r-coh — COMMENTED** [review]
Awsome collabo !!!... @zacck thanks for this.

File: test/lib/qb_backend/posts/posts_test.exs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    test "get_manual/1 returns an error is a manual does not exist" do
+        assert Repo.aggregate(Manual, :count, :id) == 0
+        assert {:error, "No Manual with id: #{@id} on the system"} == Posts.get_manual(@id)
+    end
+
+    test "create_manual/1 saves manual with valid data" do
+       profile = insert(:profile)
+       assert Repo.aggregate(Manual, :count, :id) == 0
+       assert {:ok, %Manual{} = mnl} =  Posts.create_manual(profile, @valid_manual)
```
Comment (zacck-zz) [inline test/lib/qb_backend/posts/posts_test.exs]: @br4inii  `mnl` here is not used so can be removed

**Review by zacck-zz — CHANGES_REQUESTED** [review]
Just a few changes to push this over @br4inii

**Pull request merged**